frontend/src/styles.scss

```diff
@@ -56,6 +58,7 @@ body {
   border-left: 5px solid #ae0a0a; /* red */
 }
 
-.card {
+// margin top 10
+.mt-1 {
   margin-top: 10px;
```

Comment:
We will need the .card class i think, don't know why a margin top is named "card" :p